### PR TITLE
srvr command adapt zookeeper 3.7.0 version

### PR DESCRIPTION
**Pull request by Shoothzj**
ZooKeeper 3.7.0 change the `srvr` command's output from 
`Zookeeper version: 3.4.6-1569965, built on 02/20/2014 09:09 GMT`
to
`Zookeeper version: 3.7.0-e3704b390a6697bfdf4b0bef79e3da7a4f6bac4b, built on 2021-03-17 09:46 UTC`

### CODE REVIEW

**Comment by Shoothzj** [comment]
@nemith PTAL, thanks

File: flw.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func FLWSrvr(servers []string, timeout time.Duration) ([]*ServerStats, bool) {
 
 		// determine current server
 		var srvrMode Mode
-		switch match[10] {
+		switch match[11] {
```
Comment (pmazzini) [inline flw.go:64]: Can we have a `zrVer` regexp that does not shift the groups?

Comment (Shoothzj) [reply flw.go:64]: Hello, I think we can't avoid shift if we are still using regex compile. Maybe we can swicth to parse by line ?

**Comment by Shoothzj** [comment]
@yarikk PTAL thanks

**Comment by Shoothzj** [comment]
@yarikk PTAL thanks

**Comment by nemith** [comment]
Thoughts on named groups?  Or just trying to find a different reviewer?

**Comment by nemith** [comment]
Aslo you need to rebase.

**Comment by Shoothzj** [comment]
> Thoughts on named groups? Or just trying to find a different reviewer?

@nemith  I think that on named groups, we should switch to parse line by line. Changes are a bit big

**Comment by Shoothzj** [comment]
I am not very familiar with golang. I'm sorry if this makes you think I'm not looking for code quality.
image
Looking forward to your reply.
Thanks.

**Comment by Shoothzj** [comment]
@nemith looking forward to your reply. Thanks

**Comment by Shoothzj** [comment]
@nemith Please take a look when you have time, thanks

**Comment by Shoothzj** [comment]
@nemith Please take a look when you have time, thank

**Comment by pmazzini** [comment]
@Shoothzj How about doing something like in #92

**Pull request closed without merging**